README.md

```diff
@@ -3,3 +3,8 @@ This repository contains the source files of the Uphold API documentation which
 
 # Contributing
 All submissions are welcome. To submit a change, fork this repo, commit your changes, and send us a pull request.
+
+[![Creative Commons Attribution 4.0 license][license-badge-image]](license-badge-link)
```

Comment:
Apologies, I gave you incorrect markup. The last set of brackets should also be square brackets rather than round ones, i.e.: `[license-badge-link]`